test/PProf.jl

```diff
@@ -78,4 +79,55 @@ end
     @test load_prof_proto(pprof(out=tempname(), web=false, keep_frames = "foo")).keep_frames != 0
 end
 
+@testset "@pprof macro" begin
+    # let arr = []
+        @pprof foo(10000, 5, [])
+        # sleep(2)
+    # end
+    @test PProf.proc[] !== nothing
+    @test process_running(PProf.proc[])
+
+    PProf.kill()
+    @test PProf.proc[] === nothing
+
+    @test isfile("profile.pb.gz")
+    rm("profile.pb.gz")
 end
+
+@testset "subprocess refresh" begin
+
+    let arr = []
+        @pprof foo(10000, 5, arr)
+        sleep(2)
```

Comment:
why do we need to `sleep()` in this test (here and below)? `@pprof` shouldn't return until the process is started, right?

Reply:
I just copied the thing from above in the file.

Reply:
the sleep after `refresh()` and `kill()` is actually necessary for the tests to run reliably.